src/frontends/pytorch/src/op/native_multi_head_attention.cpp

```diff
@@ -0,0 +1,192 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/opsets/opset10.hpp"
+#include "pt_framework_node.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+using namespace ov::op;
+
+OutputVector translate_native_multi_head_attention(const NodeContext& context) {
+    /*
+    aten::_native_multi_head_attention(
+        Tensor query,
+        Tensor key,
+        Tensor value,
+        int64 embed_dim,
+        int64 num_head,
+        Tensor qkv_weight,
+        Tensor qkv_bias,
+        Tensor proj_weight,
+        Tensor proj_bias,
+        Optional[Tensor] mask = None,
+        bool need_weights = true,
+        bool average_attn_weights = true,
+        Optional[int64] mask_type = None
+    )
+    */
+    num_inputs_check(context, 13, 13);
+    const auto query = context.get_input(0);
+    const auto key = context.get_input(1);
+    const auto value = context.get_input(2);
+    const auto embed_dim = context.get_input(3);
+    const auto num_head = context.get_input(4);
+    const auto qkv_weight = context.get_input(5);
+    const auto qkv_bias = context.get_input(6);
+    const auto proj_weight = context.get_input(7);
+    const auto proj_bias = context.get_input(8);
+    const auto need_weights = context.const_input<bool>(10);
+    const auto average_weights = context.const_input<bool>(11);
+
+    const auto minus_inf =
+        context.mark_node(opset10::Constant::create(element::f32, Shape{}, {-std::numeric_limits<float>::infinity()}));
+
+    const auto neg_one_1d = context.mark_node(opset10::Constant::create(element::i64, Shape{1}, {-1}));
+    const auto zero_1d = context.mark_node(opset10::Constant::create(element::i64, Shape{1}, {0}));
+    const auto one_1d = context.mark_node(opset10::Constant::create(element::i64, Shape{1}, {1}));
+    const auto two_1d = context.mark_node(opset10::Constant::create(element::i64, Shape{1}, {2}));
+    const auto three_1d = context.mark_node(opset10::Constant::create(element::i64, Shape{1}, {3}));
+    auto embed_dim_1d = context.mark_node(std::make_shared<opset10::Unsqueeze>(embed_dim, zero_1d));
+    embed_dim_1d = context.mark_node(std::make_shared<opset10::ConvertLike>(embed_dim_1d, zero_1d));
```

Comment:
Looks like this `ConvertLike` can be just `Convert, because you know the type of the constant.

Reply:
Done!